types/react-hooks-compose/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "private": true,
+    "name": "@types/react-hooks-compose",
+    "version": "2.0.9999",
+    "projects": [
+        "https://github.com/helloitsjoe/react-hooks-compose#readme"
+    ],
+    "dependencies": {
+        "@types/react": "*",
+        "react-hooks-compose": "*"
```

Comment:
This should not be here; you're writing the types for this package, and should not depend on it.

Reply:
Done, thanks